### ISSUE TITLE
bug: Mergify merged PR without required test passed

### ISSUE THREAD

**Issue by Xuanwo** [datafuselabs/databend#6982]
**Summary**

https://github.com/datafuselabs/databend/pull/6948 has been merged with `test_sqllogic_ydb_standalone_linux` failed.

Because we write `mergify.yml` in the way:

```yml
- check-success~=^test_sqllogic_(base|ydb)_standalone_linux$
```

This means check-success `base or ydb` passed, which is not expected.

We need to change this into:

```yml
- check-success=test_sqllogic_base_standalone_linux
- check-success=test_sqllogic_ydb_standalone_linux
```

cc @ZeaLoVe

**Comment by ZeaLoVe** [comment]
hmmm.. I will fix it.

**Comment by leiysky** [comment]
By the way, I've fixed the failed test case in #6979
